src/jquery.js

```diff
@@ -6,14 +6,11 @@
 
         $( function() {
 
-            $.fn.donutty = function( options ) {
+            $.fn.donutty = function() {
```

Comment:
```suggestion
            $.fn.donutty = function( options ) {
```
bring back imperative options